tests/unit/routes/test_lazy_app_container.py

```diff
@@ -121,7 +121,7 @@ def test_construction__empty_app_config_has_two_instances(app_container_construc
         **app_container_constructor_params,
         app_container_config=app_container_config,
     )
-    assert len(app_container.instance_map) == 4
+    assert len(app_container.instance_map) == 5
 
```

Comment:
If i understood this correctly, this test `test_construction__empty_app_config_has_two_instances` is to check how many instances within the empty app, right?

Reply:
yeap! this is updated because i added the `persistent` object as one of the default ones in the empty app!

Reply:
Ok. The `two_instances` in the test name seems a bit confusing to me.

Reply:
ah i probably never updated the name. i'll be making another slight tweak to this shortly, so i'll update it in a follow-up.